server/apps/users/forms.py

```diff
@@ -0,0 +1,105 @@
+from django import forms
+
+class UserProfileForm(forms.Form):
+
+    age_brackets = [
+        ("under 18", "Under 18"),
+        ("18-25", "18-25"),
+        ("26-35", "26-35"),
+        ("36-45", "36-45"),
+        ("46-65", "46-65"),
+        ("over 65", "Over 65"),
+        ("unspecified", "Prefer not to say"),
+    ]
+    age_bracket = forms.ChoiceField(choices = age_brackets, widget = forms.Select(), required=False)
+    age_bracket.group = 1
+
+    genders = [
+        ("nonbinary", "Non-binary"),
+        ("female", "Female"),
+        ("male", "Male"),
+        ("see_description", "See description"),
+        ("unspecified", "Prefer not to say"),
+    ]
+    gender = forms.ChoiceField(choices = genders, widget = forms.Select(), required=False)
+    gender.group = 1
+
+    # Taken as examples from:
+    # https://icd.who.int/browse10/2015/en#/F84
+    # See also:
+    # https://www.autism.org.uk/advice-and-guidance/topics/diagnosis/diagnostic-criteria/all-audiences
+    # Not intended as final
+    self_identifications = [
```

Comment:
https://journals.sagepub.com/doi/abs/10.1177/1362361315588200 - this article is useful as it gives advice on terminology based on consultation with autistic people. "autistic person" is preferred over "person with autism". "high" and "low" functioning are also often used to differentiate autistic people on their level of care needs. This article is useful and recommends referring "high", "medium" and "low" support needs instead: https://themighty.com/topic/autism-spectrum-disorder/autism-functioning-labels-low-functioning-high-functioning/. Also many people realise or suspect they are autistic without having a formal diagnosis.

Reply:
Thanks @GeorgiaHCA! I would suggest we start with one or two closed dropdown questions: 

1. Do you identify as autistic: Yes/No
2. Were you diagnosed as autistic?: Not yet / Yes, as a child / Yes, as an adult 

And then we have something quite simple to request feedback on next week?

Reply:
We also need to think about how we ask about connection to autism. It would be useful to know if someone is also a parent/relation/carer of an autistic person or works with autistic people professionally.

Reply:
@GeorgiaHCA Thanks, let's maybe put those suggestions into the meeting agenda for next week to discuss there?

Reply:
Thanks for the feedback on this. I've updated the field to simplify it basd on the discussion, and so that it can be considered at the next meetup. I've also created task #533 for updating the fields once they've been finalised.